src/autocomplete/NotifProvider.tsx

```diff
@@ -43,7 +43,8 @@ export default class NotifProvider extends AutocompleteProvider {
         if (!this.room.currentState.mayTriggerNotifOfType('room', client.credentials.userId)) return [];
 
         const { command, range } = this.getCurrentCommand(query, selection, force);
-        if (command && command[0] && '@room'.startsWith(command[0]) && command[0].length > 1) {
+        if (command && command[0] && ('@room'.startsWith(command[0]) || '@channel'.startsWith(command[0])) &&
```

Comment:
maybe we should have more aliases, what else do other messengers use? I know `@here` has a slightly different connotation/function but might still make a sane alias

Reply:
Slack and Discord both have an `@everyone` as well. For Slack that would be like @-ing an entire Space but Discord seems to use it more like `@room`.

I suppose it doesn't hurt to suggest those for now. If real `@here` or `@everyone` functionality gets added to Element in the future the suggestions can be repurposed for those functions.